addons/website_theme_install/static/src/less/website_theme_install.less

```diff
@@ -1,42 +1,173 @@
+@o-theme-kanban-gray: #fcfcfc;
+
 .o_kanban_view.o_theme_kanban {
+    background-color: @o-theme-kanban-gray;
+
     &.o_kanban_ungrouped {
         .o-justify-content(center);
 
         .o_kanban_record {
-            .o-flex(0, 0, auto);
-            width: 100%;
-            @media (min-width: @screen-sm-min) {
-                width: 45%;
+            .o-flex(0, 1, auto);
+            width: 250px;
+
+            &:first-child, &:nth-child(2) {
+                @media (max-width: @screen-xs-max) {
+                    margin-top: 10px;
+                }
             }
         }
     }
 
     .o_kanban_record {
-        padding: 0;
+        margin: 1% 2% 2%;
+        box-shadow: none;
+        background: none;
+        border: none;
+
+        .o_browser_bar:before {
+            content: "•••";
+            display: block;
+            padding-left: 10px;
+            color: darken(@o-theme-kanban-gray, 10%);
+            font: 12px/20px Courier, monospace;
+            letter-spacing: -1px;
```

Comment:
Mmh, in this case ok. But I already had issues reported by clients because of this rule ;)